src/ExportedFunctions/Invoke-MSTerminalGif.ps1

```diff
@@ -0,0 +1,98 @@
+
+function Invoke-MSTerminalGif {
+    <#
+.SYNOPSIS
+    Plays a gif from a URI to the terminal. Useful when used as part of programs or build scripts to show "reaction gifs" to the terminal to events.
+.DESCRIPTION
+    This command plays animated GIFs on the Windows Terminal. It performs the operation in a background runspace and only allows one playback at a time. It also remembers your previous windows terminal settings and puts them back after it is done
+.EXAMPLE
+    PS C:\> Invoke-TerminalGif https://media.giphy.com/media/g9582DNuQppxC/giphy.gif
+    Triggers a gif in the current Windows Terminal
+.EXAMPLE
+    PS C:\> Invoke-TerminalGif (Search-Giphy -DirectUrl)
+    Triggers a gif in the current Windows Terminal
+.NOTES
+    Requires Windows Terminal Preview
+#>
+    [CmdletBinding()]
+    param (
+        #The URI of the GIF you want to display
+        [Parameter(Mandatory)][uri]$Uri,
+        #The name or GUID of the Windows Terminal Profile in which to play the Gif. If not specified, attempts to intelligently determine it from the executable name or if $ENV:WT_PROFILE was specified by adding -noexit -command \". {$env:WT_PROFILE='myprofilename'}\" to your terminal profile definition.
```

Comment:
I don't see where it checks $ENV:WT_PROFILE

Reply:
That is in the next pull request that will automatically detect "this" profile if the command is run inside windows terminal, I will fix it for clarification.